src/app/search/search-filters.ts

```diff
@@ -1153,25 +1153,13 @@ function searchFilters(
         );
       },
       event(item: D2Item, predicate: string) {
-        if (!item || !item.source) {
+        if (!item || !item.source || !item.event) {
           return false;
         }
-        let searchedEvent = 0;
-        switch (predicate) {
-          case 'dawning':
-            searchedEvent = 1;
-            break;
-          case 'crimsondays':
-            searchedEvent = 2;
-            break;
-          case 'solstice':
-            searchedEvent = 3;
-            break;
-          case 'fotl':
-            searchedEvent = 4;
-            break;
-        }
-        return _.includes(D2Events[predicate].source, item.source) || searchedEvent === item.event;
+        return (
+          _.includes(D2Events[predicate].source, item.source) ||
```

Comment:
Any reason to use `_.includes` instead of just `D2Events[predicate].includes(item.source)`?

Reply:
It did not want to work for me when I initially added `sources:`, but it is working now so pushed that change for both `sources` and `events`.